routes/setup.js

```diff
@@ -5,6 +5,7 @@ const models = require("../db/models");
 const routeUtils = require("./utils");
 const constants = require("../data/constants");
 const roleData = require("../data/roles");
+const modifierData = require("../data/modifiers");
```

Comment:
he should have added another api like /getModifiers to get all 

and also to load them in siteinfo in Main.jsx

Reply:
Added to Main.jsx